src/datasets/tasks/question_answering.py

```diff
@@ -0,0 +1,40 @@
+from typing import Dict
+
+from ..features import Features, Value
+from .base import TaskTemplate
+
+
+class QuestionAnswering(TaskTemplate):
+    task = "question_answering"
+    input_schema = Features({"question": Value("string"), "context": Value("string")})
+    label_schema = Features({"answer_start": Value("int32"), "answer_end": Value("int32")})
```

Comment:
In general we only have `answer_text` and `answer_start`, not `answer_end` and `answer_start`

Reply:
good catch!